integration/manifests/svcwatcher/0svcwatcher_rbac.yaml

```diff
@@ -33,6 +33,12 @@ rules:
   - list
   - watch
   - get
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
```

Comment:
when we fixed -in another :) - upstream we found we need to add more verbs for the leader election library to work as intended:
https://gerrit.akraino.org/r/c/ta/caas-security/+/2084/4/rbac_manifests/svcwatcher-rbac-config.yml

Reply:
Added update and get here, too